### ISSUE TITLE
Backup best practice

### ISSUE THREAD

**Issue by zimski** [vitabaks/autobase#782]
From the first time I used the solution, I noticed that for Hetzner, no backup was installed because the provider doesn't support S3.

I get it, but :) :
1. I think it's a good practice to encourage the users to backup the database outside of the provider that hosts the cluster
2. Now Hetzner supports S3 (Beta for now)

The 1. implies that we need a form when setting up the cluster to set up the S3 credential and endpoint.
The advantage is that this allows the user to host the backup wherever they want, ideally with a different provider.

If this aligns with your vision of the product I can prioritize some times to do the dev as we need this to go into production with your solution.

**Comment by vitabaks** [comment]
I completely agree with you. We plan to add to the “[Expert mode](https://github.com/vitabaks/postgresql_cluster/issues/756)” the ability to configure backup settings to be able to specify the necessary options (e.q., AWS S3).

This can be done via the [command line](https://postgresql-cluster.org/docs/management/backup#command-line), but not yet in the UI.

**Comment by vitabaks** [comment]
> Now Hetzner supports S3 (Beta for now)

Thanks, I'll take a look at it.
